modules/govuk/lib/puppet/parser/functions/data_sync_times.rb

```diff
@@ -0,0 +1,38 @@
+module Puppet::Parser::Functions
+  newfunction(:data_sync_times, :type => :rvalue, :doc => <<-EOS
```

Comment:
I wonder if you want to name this as `format_data_sync_time` since the primary role of the function is the formatting?

Reply:
Since the function returns values and doesn't change anything, I'd probably avoid the use of a verb like "format" in its name. Calling it `format_data_sync_time` makes it sound like it's either editing something somewhere or returning edited versions of passed attributes. I think `data_sync_times` is accurate for what it does: returning appropriate data sync time values based on the argument passed.

Reply:
I think I agree with Alan here 👍